app/Console/Commands/UpdateFiltersDatasetGat5632.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace App\Console\Commands;
+
+use Illuminate\Console\Command;
+use App\Models\Filter;
+
+class UpdateFiltersDatasetGat5632 extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'app:update-filters-dataset-gat5632';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Renames containsTissue filter to containsBioSamples';
+
+    /**
+     * Execute the console command.
+     */
+    public function handle()
+    {
+        Filter::updateOrCreate([
+            'type' => 'dataset',
+            'keys' => 'containsTissue',
+        ], [
+            'type' => 'dataset',
+            'keys' => 'containsBioSamples',
+        ]);
+    }
```

Comment:
feels like if this is changing filters - there needs to be something to trigger a re-index of anything this effects?

Been a while, so would check with @BSnelling @spco